src/org/jitsi/impl/neomedia/rtp/RTPEncodingDesc.java

```diff
@@ -452,8 +452,10 @@ void update(RawPacket pkt, long nowMs)
         long ts = pkt.getTimestamp();
         FrameDesc frame = frames.get(ts);
 
+        boolean isNew;
```

Comment:
could we rename to something like `isFirstPacketOfNewFrame` maybe?  also, if we initialize it to false we can get rid of the else block below